src/logicblocks/event/store/adapters/memory/converters.py

```diff
@@ -18,41 +18,43 @@
 from logicblocks.event.store.exceptions import UnmetWriteConditionError
 from logicblocks.event.types import (
     Converter,
-    Event,
     StoredEvent,
     StreamIdentifier,
 )
 
 from .db import InMemoryEventsDBTransaction
-from .types import QueryConstraintCheck
+from .types import InMemoryQueryConstraintCheck
 
 
-class SequenceNumberAfterConstraintConverter(
-    Converter[SequenceNumberAfterConstraint, QueryConstraintCheck[Event]]
+class OrderingIdAfterConstraintConverter(
```

Comment:
I don't think I like it that this is in the store package given that it's a general purpose constraint.

Reply:
Fixed